packages/xchain-thorchain/src/client.ts

```diff
@@ -221,6 +221,14 @@ class Client implements ThorchainClient, XChainClient {
     })
   }
 
+  /**
+   * Get cosmos clinet
+   * @returns {CosmosSDKClient} current cosmos client
+   */
+  getCosmosClient = (): CosmosSDKClient => {
```

Comment:
Add `getCosmosClient` to `interface ThorchainClient`

Reply:
https://github.com/xchainjs/xchainjs-lib/pull/306/commits/b61a74dfdbecf6d53e5e4f3bcd0e6b19bcc2c0cd